src/static/current-state/components/tabs/SystemTab.tsx

```diff
@@ -1,17 +1,11 @@
 import * as React from 'react';
 import { TabContent } from './TabContent';
-import { User } from '../User';
-import { PermissionControl } from '../PermissionControl';
-
-// sort by permssion descending (admin -> land -> read)
-function sortUsersByPermission(user1: IPermission, user2: IPermission) {
```

Comment:
Where does it go?

Reply:
This is moved into `src/static/current-state/components/tabs/AllowedUsers.tsx`